Sources/Nophan/Debug/NophanMonitor.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  NophanMonitor.swift
+//  
+//
+//  Created by Usman Nazir on 28/05/2024.
+//
+
+import Foundation
+
+/// A struct that maintains an array of the last 15 NophanRequest objects.
+internal class NophanMonitor: ObservableObject {
```

Comment:
Why are you using `ObservableObject` instead of `@Observable`

Reply:
yes

Reply:
We use `@Observable` on Feast. Will Nophan be used on the main app as well? If yes then nevermind.